### ISSUE TITLE
Error: 'Page' is imported from both pages.dart and navigator.dart

### ISSUE THREAD

**Issue by madebycm** [xsahil03x/fancy_on_boarding#32]
```
Compiler message:
../../flutter/.pub-cache/hosted/pub.dartlang.org/fancy_on_boarding-1.5.0/lib/src/fancy_on_boarding.dart:67:9: Error: 'Page' is imported from both 'package:fancy_on_boarding/src/pages.dart' and 'package:flutter/src/widgets/navigator.dart'.
        Page(
        ^^^^
../../flutter/.pub-cache/hosted/pub.dartlang.org/fancy_on_boarding-1.5.0/lib/src/fancy_on_boarding.dart:73:18: Error: 'Page' is imported from both 'package:fancy_on_boarding/src/pages.dart' and 'package:flutter/src/widgets/navigator.dart'.
          child: Page(
                 ^^^^
```

Steps to reproduce:

```
import "package:flutter/material.dart";
import 'package:fancy_on_boarding/fancy_on_boarding.dart' as boarder;

final pageList = [
    boarder.PageModel(
        color: const Color(0xFF678FB4),
        heroAssetPath: 'assets/heart.png',
        title: Text('Hotels',
            style: TextStyle(
              fontWeight: FontWeight.w800,
              color: Colors.white,
              fontSize: 34.0,
            )),
        body: Text('All hotels and hostels are sorted by hospitality rating',
            textAlign: TextAlign.center,
            style: TextStyle(
              color: Colors.white,
              fontSize: 18.0,
            )),
        iconAssetPath: 'assets/heart.png'),
    boarder.PageModel(
        color: const Color(0xFF65B0B4),
        heroAssetPath: 'assets/heart.png',
        title: Text('Banks',
            style: TextStyle(
              fontWeight: FontWeight.w800,
              color: Colors.white,
              fontSize: 34.0,
            )),
        body: Text(
            'We carefully verify all banks before adding them into the app',
            textAlign: TextAlign.center,
            style: TextStyle(
              color: Colors.white,
              fontSize: 18.0,
            )),
        iconAssetPath: 'assets/heart.png'),
    boarder.PageModel(
      color: const Color(0xFF9B90BC),
      heroAssetPath: 'assets/heart.png',
      title: Text('Store',
          style: TextStyle(
            fontWeight: FontWeight.w800,
            color: Colors.white,
            fontSize: 34.0,
          )),
      body: Text('All local stores are categorized for your convenience',
          textAlign: TextAlign.center,
          style: TextStyle(
            color: Colors.white,
            fontSize: 18.0,
          )),
      iconAssetPath: 'assets/heart.png',
    ),
];


class Onboarding extends StatelessWidget {

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: boarder.FancyOnBoarding(
        doneButtonText: "Done",
        skipButtonText: "Skip",
        pageList: pageList,
        onDoneButtonPressed: () =>
            Navigator.of(context).pushReplacementNamed('/mainPage'),
        onSkipButtonPressed: () =>
            Navigator.of(context).pushReplacementNamed('/mainPage'),
      ),
    );
  }
}
```

Running **Flutter (Channel beta, v1.17.0, on Mac OS X 10.15.4 19E287, locale en-NO)**

**Comment by xsahil03x** [comment]
@madebycm you just need to hide the `Page` class from `fancy_on_boarding`.

So instead of
```import 'package:fancy_on_boarding/fancy_on_boarding.dart' as boarder;```

Write
```import 'package:fancy_on_boarding/fancy_on_boarding.dart' hide Page;```

**Comment by AlpenYeti** [comment]
@xsahil03x This fix does not seem to work as Page is not directly imported from `package:fancy_on_boarding/fancy_on_boarding.dart` but from `fancy_on_boarding/src/pages.dart`. Therefore, adding the `hide Page` bit does not prevent the conflict.

Would you have any further recommendation on how to move forward with this, as this onboarding plugin seems to be the most advanced and customizable so far ?

**Comment by mrpaul92** [comment]
same here... any fix available ?

**Comment by VictorUvarov** [comment]
Confirmed that the Page from the material import is conflicting with this packages definition of a page.

**Comment by xsahil03x** [comment]
The simple fix is to just rename the Page class to something else, Also there is a PR #29 for this particular fix, but I need some changes in it and he is not responding. I might close that and do it myself.

**Comment by VictorUvarov** [comment]
I wouldn't wait since your package is now broken

**Comment by xsahil03x** [comment]
Fixed in v1.6.0

**Comment by syedanwarafridi** [comment]
package:flutter/src/widgets/navigator.dart

Click on the upper link. Find all 'Page' in the dart file and replace that 'Page' with 'Pagee' (Change the Spelling). and save the code and execute the file. You will not face this error again.
